Dockerfile.builder

```diff
@@ -59,3 +57,13 @@ RUN wget --quiet -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | ap
     apt-get update && \
     apt-get install --no-install-recommends -y -q \
     google-chrome-stable ca-certificates
+
+LABEL \
+        io.k8s.description="This is a component of OpenShift Container Platform and provides a web console." \
+        com.redhat.component="openshift-enterprise-console-container" \
+        maintainer="Samuel Padgett <spadgett@redhat.com>" \
+        name="openshift3/ose-console" \
```

Comment:
This seems wrong since it's OpenShift 4 now.

Reply:
Right, I didn't update anything yet per my comment, do we need all these labels for this or shall we skip?